src/main/java/app/model/SupplyFruitBatch.java

```diff
@@ -0,0 +1,52 @@
+package app.model;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+public class SupplyFruitBatch {
```

Comment:
batch means several items. Consider renaming
```suggestion
public class Fruit {
```